playground/static-site/package.json

```diff
@@ -1,20 +1,32 @@
 {
   "name": "static-site",
-  "private": true,
-  "version": "1.0.0",
+  "version": "0.0.0",
+  "description": "An example repository to help users get started with React development on Yext Pages",
+  "author": "sumo@yext.com",
+  "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "pages dev --local",
+    "dev": "pages dev --local --no-open-browser",
```

Comment:
can we add `--no-prod-url` here so we don't get the logs about being unable to generate pages for documents without slugs?

Reply:
I can just remove the currently unused localData which would probably be cleaner

Reply:
I'd rather keep at least one entity file in localData since I was imagining we'd just add an entity page test to static-site (and rename it). That way, we won't need another test repo for an entity page, and only a second one for multibrand

Reply:
I was thinking we can do that when the entity page test is added? since it would be unused in this PR